__tests__/product/product.query.test.js

```diff
@@ -1,10 +1,16 @@
 /* eslint-disable no-undef */
 const { gql } = require('apollo-boost');
 const client = require('../../utils/apollo-test-client');
-const { newModel, badProductId, newCategory, getNewProduct } = require('./product.variables');
+const {
+  newModel,
+  badProductId,
+  newCategory,
+  getNewProduct,
+} = require('./product.variables');
 require('dotenv').config();
 
-let productId, categoryId, subcategoryId, modelId
+let productId; let categoryId; let subcategoryId; let
+    modelId;
```

Comment:
Is this ESlint or you? Please change it back

Reply:
it is formatted by eslint